06-statistical-testing.Rmd

```diff
@@ -621,7 +645,7 @@ H_0: p_{1_{Biden}} &= p_{1_{Trump}} = p_{1_{Other}},\\
     p_{5_{Biden}} &= p_{5_{Trump}} = p_{5_{Other}},\\
     p_{6_{Biden}} &= p_{6_{Trump}} = p_{6_{Other}}
   \end{align*}
-    where $p_{i_{Biden}}$ is the observed proportion of each age group ($i$) that voted for Biden, $p_{i_{Trump}}$ is the observed proportion of each age group ($i$) that voted for Trump, and $p_{i_{Other}}$ is the observed proportion of each age group ($i$) that voted for another candidate
+    where $p_{i_{Biden}}$ is the observed proportion of each age group ($i$) that voted for President Joseph Biden, $p_{i_{Trump}}$ is the observed proportion of each age group ($i$) that voted for President Donald Trump, and $p_{i_{Other}}$ is the observed proportion of each age group ($i$) that voted for another candidate
```

Comment:
```suggestion
    where $p_{i_{Biden}}$ is the observed proportion of each age group ($i$) that voted for Joseph Biden, $p_{i_{Trump}}$ is the observed proportion of each age group ($i$) that voted for Donald Trump, and $p_{i_{Other}}$ is the observed proportion of each age group ($i$) that voted for another candidate
```

Reply:
At the time of the voting, Biden wasn't president, so it would have been "Former Vice President Joseph Biden" and "President Donald Trump".  I think to avoid confusion we stick with just their names and not their titles.